test/blueprints/Do.Test.Blueprints.Service.Test/TestServiceSpec.cs

```diff
@@ -5,6 +5,14 @@ public abstract class TestServiceSpec : ServiceSpec
     static TestServiceSpec() =>
         Init(
             business: c => c.Default(assemblies: [typeof(Entity).Assembly]),
+            communication: c => c.Mock(configurationBuilder =>
+            {
+                configurationBuilder.AddClientSetup<Singleton>([new(r => true, "test result")]);
```

Comment:
create an overload for the constructor to enable `new("test result")` without being have to specify `r => true`